scripts/upgrade-zulip-from-git

```diff
@@ -33,11 +34,16 @@ if os.getuid() != 0:
     logging.error("Must be run as root.")
     sys.exit(1)
 
-if len(sys.argv) != 2:
-    print(FAIL + "Usage: upgrade-zulip-from-git refname" + ENDC)
-    sys.exit(1)
+parser = argparse.ArgumentParser()
+parser.add_argument("refname", help="Git reference.")
+parser.add_argument("--remote-url", dest="remote_url", help="Git remote URL.")
+args = parser.parse_args()
```

Comment:
You might want to have a look at the descriptions, let me know if anything needs to be changed.